src/plugins/condition/components/DropHint.vue

```diff
@@ -0,0 +1,83 @@
+/*****************************************************************************
+ * Open MCT, Copyright (c) 2014-2018, United States Government
```

Comment:
Change year to `2020`

Reply:
DropHint.vue isn't used and shouldn't have been part of this PR. I've removed it. I went ahead and added the current legal comment header for all vue components in condition directory